Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -86,34 +103,53 @@ void ApplyRenderSettings()
 	void LerpRenderSettings( RenderSettingsData initRenderSettings, float delta )
```

Comment:
It's used in the coroutine at the end of that script. This is used by render volumes